My_Package/Subtract_Pack/Subtract_Pack.py

```diff
@@ -0,0 +1,4 @@
+def subtract(num1,num2):
+	print("I am in Subtract package.")
+	result=num1-num2
```

Comment:
This trial comment

Reply:
.pyc removed and ignored using .gitignore file